crypto/dsa/dsa.go

```diff
@@ -0,0 +1,87 @@
+package dsa
+
+import (
+	"fmt"
+
+	"github.com/tbd54566975/web5-go/crypto/dsa/ecdsa"
+	"github.com/tbd54566975/web5-go/crypto/dsa/eddsa"
+	"github.com/tbd54566975/web5-go/jwk"
+)
+
+var AlgorithmID = struct {
```

Comment:
> It would also be more typical to prefix the name with the type, so that they're grouped together in docs and by autocompletion

good point! i originally had it the way you suggest, but ran into a cyclic import issue because i put `type AlgorithmID string` in the `dsa` package and was importing it into `ecdsa` and `eddsa`. But then `dsa.go` imports `ecdsa` and `eddsa`. 

i suppose i could just ditch the type alias or restructure the package

Reply:
done: https://github.com/TBD54566975/web5-go/commit/893cc4e0f6b983ab1982a2976858337a5f364707

decided to leave package restructuring for a follow-up PR if desired. i'll need to adopt an interface + registration pattern in order to sidestep the cyclic import issue described above